src/extractors/exported.js

```diff
@@ -163,17 +186,83 @@ function traverseExportedSubtree(path, data, addComments, overrideName) {
     target = target.get('init');
   }
 
+  if (t.isTSInterfaceDeclaration(target)) {
+    addComments(data, path, undefined, overrideMemberOf);
+
+    // Heritage needs documentation too
+    if (target.node.extends) {
+      target.node.extends
+        .map(({ expression }) => expression.name)
+        .forEach(name => {
+          const binding = getBinding(path.scope, name);
+          traverseExportedSubtree(
+            binding,
+            data,
+            addComments,
+            overrideName,
+            overrideMemberOf
+          );
+        });
+    }
+  }
+
   if (target.isClass() || target.isObjectExpression()) {
+    // Handle mixins if a declaration is found in the target scope
+    const declaration = findDeclaration(
+      path.scope,
+      target.node.id.name,
+      'TSInterfaceDeclaration'
+    );
+    if (declaration)
+      declaration.node.extends
+        .map(({ expression }) => getBinding(path.scope, expression.name))
+        .filter(binding => binding)
+        .forEach(binding => {
+          traverseExportedSubtree(
+            binding,
+            data,
+            addComments,
+            overrideName,
+            target.node.id.name
+          );
+        });
     target.traverse({
       Property(path) {
         addComments(data, path);
+        // If leading comment has @inner tag
+        if (
+          path.node.leadingComments &&
+          path.node.leadingComments
+            .map(({ value }) => value.match(/^ *\*\*? ?@inner/))
```

Comment:
This regexp should be declared somewhere outside and given a readable name (`IS_INNER_RE`)